lib/crawler/keyword/keywords.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Crawler.Keyword.Keywords do
+  @moduledoc """
+  The Keywords context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias Crawler.Keyword.Queries.KeywordQuery
+  alias Crawler.Keyword.Schemas.Keyword
+  alias Crawler.Repo
+
+  def list_keywords(user_id), do: Repo.all(KeywordQuery.user_keywords_query(user_id))
+
+  def create_keyword(attrs \\ %{}) do
+    %Keyword{}
+    |> Keyword.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def create_keyword_list(keyword_list, user_id) do
+    keyword_params_list = build_keyword_params_list(keyword_list, user_id)
+
+    insert_keywords(keyword_params_list, user_id)
+  end
+
+  defp insert_keywords(keyword_params_list, user_id) do
+    Enum.each(keyword_params_list, fn keyword_params ->
+      keywords = Repo.all(KeywordQuery.user_keywords_query(keyword_params.name, user_id))
```

Comment:
@andyduong1920 I still go with option 2. But instead of `Repo.exist`, I will use `Repo.one`. One more time, I need clarification on the performance between them. Any suggestions, please?

Reply:
> But instead of Repo.exist, I will use Repo.one

You can check the different between `Repo.exist?` and `Repo.one` @vnntsu 

- Repo exists -> just return true/false, it doesn;t load all attributes from the DB -> and the select SQL query is simpler 
- Repo.one -> it query and load all attributes from the DB and it return `an object`


So perf: ➡️ Repo.exist?

Reply:
@andyduong1920 Thanks for the explanation 🙏 . In my case, I'd like to retrieve the DB record, because I will update the status to `new` later.